src/mmu/alloc.c

```diff
@@ -37,27 +37,41 @@ int liballoc_unlock() {
 
 void* liballoc_alloc(int number_of_pages) {
     uint64_t first_free_page = 0;
+    uint32_t counter = 0;
 
     for (uint64_t i = 1; i <= max_pages; i++) {
         if (bitmap_get(allocated_pages, i) == false) {
-            first_free_page = i;
-            break;
+
```

Comment:
Sorry, I don't like empty lines everywhere :) Unfortunately, `astyle` cannot enforce all my coding standards for this project. I guess, not having too many empty lines would be good, like no empty lines after opening blocks, etc.

Reply:
My apologies, I'm used to how we do things at my job, which I feel helps code readability quite a lot. I'll follow your standards better over time.